src/API/Router.test.ts

```diff
@@ -4,6 +4,9 @@ import * as Pino from 'pino'
 import { describe } from 'riteway'
```

Comment:
This is unrelated to your PR, but can you just delete this file?

We introduced a bunch of fake tests just to hack our way around coverage a while back and decided it was a bad idea quickly afterwards. 

I keep seeing PRs having to deal with these fake tests to keep the code passing. Better to just delete them IMO.

@kennylavender FYI